components/grid-col/README.md

```diff
@@ -9,9 +9,4 @@ GridCol
 
 
 
-### Properties
-Prop | Required | Default | Type | Description
-:--- | :------- | :------ | :--- | :----------
- `children` |  | undefined | node | 
```

Comment:
maybe leave this in too - altho arguably it's implicit that this is where the content goes for the grid column